app/assets/javascripts/discourse/app/components/user-info.js

```diff
@@ -12,6 +12,16 @@ export default Component.extend({
   includeLink: true,
   includeAvatar: true,
 
+  didInsertElement() {
+    this._super(...arguments);
+    this.user?.trackStatus?.();
```

Comment:
Is there a difference between calling `.trackStatus()` and `.trackStatus?.()`?

Reply:
There is difference, yes. `trackStatus()` is defined on the user model, but sometimes we don't wrap a user object received from the server into a user model. If it's not a user model, but just a plain js object with user data, it won't have `trackStatus` method, so calling `this.user?.trackStatus.()` will lead to an exception. Calling `.trackStatus?.()` will help to avoid an exception, but the downside will be is that the status won't be receiving live updates.

We use this `user-info` component in several places and in some of them we don't wrap server responses into user models, but we also don't need live status updates there. So calling `this.user?.trackStatus?.();` helps support both cases.